### ISSUE TITLE
Strategic Plan Release 2024 Coordination

### ISSUE THREAD

**Issue by tonyfast** [materialsgenomefoundation/strategic-plan#15]

**Comment by tonyfast** [comment]
before we make a bunch of issues, we should decide what is a good template even if we can't use it directly. https://github.com/materialsgenomefoundation/strategic-plan/blob/main/.github/ISSUE_TEMPLATE/strategic-plan-editorial.md

the template is a starting point, and as we work on them we'll sort out some collaborative privileges.

**Comment by jaredkosters** [comment]
test

**Comment by tonyfast** [comment]
- [ ] assess each subsection
- [ ] add content, ideas, bullets

**Comment by jforshey** [comment]
Hi Tony!

**Comment by tonyfast** [comment]
@jaredkosters @jforshey glad y'all could join the party! :tada:

**Comment by jforshey** [comment]
> @jaredkosters @jforshey glad y'all could join the party! 🎉

Yeah! Can you add us as collaborators, or whatever level of access allows us to interact with the checkboxes and edit notes?

**Comment by tonyfast** [comment]
i updated your perms. can you let me know if you can perform the activities you want to?

**Comment by jforshey** [comment]
> i updated your perms. can you let me know if you can perform the activities you want to?

Not yet. When I try to interact with a checkbox on this page, for example: https://github.com/materialsgenomefoundation/strategic-plan/issues/17 , the checkbox doesn't toggle.

**Comment by jaredkosters** [comment]
> > i updated your perms. can you let me know if you can perform the activities you want to?
> 
> Not yet. When I try to interact with a checkbox on this page, for example: #17 , the checkbox doesn't toggle.

Can confirm—doesn't toggle for me either.

**Comment by tonyfast** [comment]
try again. check your emails for a confirmation message about elevated permissions.

**Comment by asaku-nexight** [comment]
Hey Tony, could you add me as a collaborator?

**Comment by tonyfast** [comment]
@asaku-nexight  done
